test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreMvcTestBase.cs

```diff
@@ -129,11 +129,11 @@ public void RunTraceTestOnSelfHosted(string packageVersion)
             string resourceUrl,
             Func<MockTracerAgent.Span, List<string>> additionalCheck = null)
         {
-            var expectation = new AspNetCoreMvcSpanExpectation(EnvironmentHelper.FullSampleName, TopLevelOperationName)
+            var resourceName = $"{httpMethod.ToUpper()} {resourceUrl}";
```

Comment:
Following the code path where  `resourceName` is used, I see that, in `SpanExpectation`'s constructor converts it to lowercase. Why do you need to do the uppercase conversion here?